GateWatcher/aioniq_ecs/ingest/parser.yml

```diff
@@ -0,0 +1,768 @@
+        filter: ''
+        name: set
+  ecs:
+    actions:
+      - set:
+          ecs.version: '{{json_load.message.ecs.version}}'
+        filter: ''
+        name: set
+  user_agent:
+    actions:
+      - set:
+          user_agent.name: '{{json_load.message.user_agent.name}}'
+          user_agent.os.name: '{{json_load.message.user_agent.os.name}}'
+          user_agent.version: '{{json_load.message.user_agent.version}}'
+          user_agent.original: '{{json_load.message.user_agent.original}}'
+          user_agent.os.family: '{{json_load.message.user_agent.os.family}}'
+          user_agent.os.version: '{{json_load.message.user_agent.os.version}}'
+          user_agent.device.name: '{{json_load.message.user_agent.device.name}}'
+          gatewatcher.user_agent.major: '{{json_load.message.user_agent.major}}'
+          gatewatcher.user_agent.minor: '{{json_load.message.user_agent.minor}}'
+          gatewatcher.user_agent.patch: '{{json_load.message.user_agent.patch}}'
+          gatewatcher.user_agent.os.major: '{{json_load.message.user_agent.os.major}}'
+        filter: ''
+        name: set
+  version:
+    actions:
+      - set:
+          gatewatcher.version: '{{json_load.message["@version"]}}'
+        filter: ''
+        name: set
+  tls:
+    actions:
+      - set:
+          tls.version: '{{json_load.message.tls.version}}'
+          gatewatcher.tls: '{{json_load.message.tls}}'
+          tls.server.issuer: '{{json_load.message.tls.server.issuer}}'
+          tls.server.subject: '{{json_load.message.tls.server.subject}}'
+          tls.server.hash.md5: '{{json_load.message.tls.server.hash.md5}}'
+          tls.server.hash.sha1: '{{json_load.message.tls.server.hash.sha1}}'
+          tls.client.server_name: '{{json_load.message.tls.client.server_name}}'
+          tls.server.certificate: '{{json_load.message.tls.server.certificate}}'
+          tls.server.hash.sha256: '{{json_load.message.tls.server.hash.sha256}}'
+          tls.server.certificate_chain: '{{json_load.message.tls.server.certificate.chain}}'
+        filter: ''
+        name: set
+  metadata-flowbits:
+    actions:
+      - set:
+          gatewatcher.metadata.flowbits: '{{json_load.message.metadata.flowbits}}'
+        filter: ''
+        name: set
+  smtp:
+    actions:
+      - set:
+          gatewatcher.smtp.helo: '{{json_load.message.smtp.helo}}'
+          gatewatcher.smtp.rcpt_to: '{{json_load.message.smtp.rcpt_to}}'
+          gatewatcher.smtp.mail_from: '{{json_load.message.smtp.mail_from}}'
+        filter: ''
+        name: set
+  email:
+    actions:
+      - set:
+          email.subject: '{{json_load.message.email.subject}}'
+          email.message_id: '{{json_load.message.email.message_id}}'
+          email.attachments: '{{json_load.message.email.attachments}}'
+          gatewatcher.email.to: '{{json_load.message.email.to}}'
+          gatewatcher.email.from: '{{json_load.message.email.from}}'
```

Comment:
These two information can be put in ECS fields `email.to.address` and `email.from.address`?